src/components/Dialog-Window/Resources/DialogResources.tsx

```diff
@@ -49,16 +49,28 @@ export const DialogResources: React.FC<DialogResourceProps> = ({
     return `https://${linkPath}`;
   };
 
+  const getRootUrl = (linkPath: string): string => {
+    const normalizedLink = normalizeLinkPath(linkPath);
+    const url = new URL(normalizedLink);
+
+    let rootUrl = url.hostname;
+    if (url.hostname.startsWith("www.")) {
+      const splitPrefix = rootUrl.concat().split("www.")[1];
+      rootUrl = splitPrefix;
+    }
+    return rootUrl;
```

Comment:
This could be simplified

```suggestion
    let rootUrl = url.hostname;
    if (rootUrl.startsWith("www.")) {
      rootUrl.replace("www.", "");
    }
    return rootUrl;
```